### PR TITLE
feat: add tabs

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR adds a new file "Application.tsx", which is analogous to the existing Project.tsx file in that it contains the base layout for the application page, as well as tabs pointing to sub pages. Currently, the overview tab uses a paragraph with some fallback text, while the connected instances table displays the instances table. 

I have mostly copied the existing ApplicationEdit component and used that as a base, assuming that we'll delete that component when we're done with this.

image

### CODE REVIEW

File: frontend/src/component/application/Application.tsx
Comment (thomasheartman) [inline frontend/src/component/application/Application.tsx:1]: This is pretty much a direct port of the old application screen, except I have ripped out the content. So what remains is the header and the page layout. 

If the sdkReporting flag is turned off, we'll show the old screen instead.

File: frontend/src/component/menu/routes.ts [hunk 1]
```diff
@@ -163,12 +163,13 @@ export const routes: IRoute[] = [
 
     // Applications
     {
-        path: '/applications/:name',
+        path: '/applications/:name/*',
```
Comment (thomasheartman) [inline frontend/src/component/menu/routes.ts:166]: We need to render for all subpaths of applications/:name too.

File: frontend/src/component/application/Application.tsx
Comment (thomasheartman) [reply frontend/src/component/application/Application.tsx:1]: I know the sketches for the header are slightly different than the existing design, but I suggest we do the updates for that in a separate PR.

**Review by sjaanus — APPROVED** [review]
Approving for now, we can change UX stuff later.

**Pull request merged**